index.md

```diff
@@ -1,27 +1,84 @@
+* Replicated Volumes as the name suggests, are those that have their data synchronously replicated to multiple nodes. Volumes can sustain node failures. The replication also can be setup across availability zones helping applications move across availability zones. OpenEBS offers MayaStor as an replicated storage solution, which provides high availability and high performance.
+
+## Documentation and user guides
+
+You can run OpenEBS on any Kubernetes 1.23+ cluster in a matter of minutes. See the [Quickstart Guide to OpenEBS](https://openebs.io/docs/quickstart-guide/installation) for detailed instructions.
+
+## Getting started
+
+### How to customize OpenEBS Helm chart?
+
+OpenEBS helm chart is an umbrella chart that pulls together engine specific charts. The engine charts are included as dependencies. 
+arts/openebs/Chart.yaml). 
+
+```bash
+openebs
+├── (default) LocalPV HostPath
+├── (default) LocalPV LVM
+├── (default) LocalPV LVM
+└── (default) MayaStor (replicated)
+```
 
 ### Prerequisites
 
- - Make sure the [system requirement pre-requisites](https://mayastor.gitbook.io/introduction/quickstart/prerequisites) are met.
- - Label the nodes same as the mayastor.nodeSelector in values.yaml
+- [LocalPV Hostpath Prerequisites](https://openebs.io/docs/user-guides/local-storage-user-guide/local-pv-hostpath/hostpath-installation#prerequisites)
+- [LocalPV LVM Prerequisites](https://openebs.io/docs/user-guides/local-storage-user-guide/local-pv-lvm/lvm-installation#prerequisites)
+- [LocalPV ZFS Prerequisites](https://openebs.io/docs/user-guides/local-storage-user-guide/local-pv-zfs/zfs-installation#prerequisites)
+- [Replicated Engine Prerequisites](https://openebs.io/docs/user-guides/replicated-storage-user-guide/rs-installation#prerequisites)
+
+### Setup Helm Repository
+
+Before installing OpenEBS Helm charts, you need to add the [OpenEBS Helm repository](https://openebs.github.io/openebs) to your Helm client.
 
-### Add the repository
+#### Setup helm repository
 
 ```bash
-helm repo add mayastor https://openebs.github.io/mayastor-extensions/ 
+helm repo add openebs https://openebs.github.io/openebs
+helm repo update
 ```
 
-### Install Mayastor
+#### Install OpenEBS helm chart with default values.
 
 ```bash
-helm install mayastor mayastor/mayastor -n mayastor --create-namespace
+helm install openebs --namespace openebs openebs/openebs --create-namespace
 ```
 
-#### ⚠ WARNING: non-production playground testing with no default storage class and single-replica etcd
+The above commands will install OpenEBS LocalPV Hostpath, OpenEBS LocalPV LVM, OpenEBS LocalPV ZFS and OpenEBS Mayastor components in openebs namespace and chart name as openebs. 
+
+If you want to not install OpenEBS Mayastor which is a replicated storage engine you can use the following command.
+
+```bash
+helm install openebs --namespace openebs openebs/openebs --set mayastor.enabled=false --create-namespace
```

Comment:
This command has changed. Also, do we want to advertise this kind of an option.

Reply:
Yeah good question, CC @avishnu

Reply:
@niladrih What do you mean by `this command has changed`?

Reply:
Ohh @niladrih You meant the condition. Yeah. Will need to update that at other places as well.

Reply:
Done.